shopify_app.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= 2.2.4"
 
   s.add_runtime_dependency('rails', '>= 4.2.6')
-  s.add_runtime_dependency('shopify_api', '~> 4.2.2')
+  s.add_runtime_dependency('shopify_api', '>= 4.2.2', '< 5.0')
```

Comment:
since its the Ruby Version that this PR fails CI, shouldn't the gemspec be:
```ruby
s.required_ruby_version = "~> 2.3"

s.add_runtime_dependency('rails', '>= 4.2.6')
s.add_runtime_dependency('shopify_api', '>= 4.2.2')
...
```